libbinaryen.opam

```
@@ -15,8 +15,8 @@ depends: [
   "conf-cmake" {build}
   "dune" {>= "3.0.0"}
   "dune-configurator" {>= "3.0.0"}
-  "js_of_ocaml-compiler" {with-test & >= "4.1.0" < "6.0.0"}
-  "ocaml" {>= "4.12"}
+  "js_of_ocaml-compiler" {with-test & >= "4.1.0" <= "6.0.1"}
```

If we support v6 now then this should be all of v6:
```suggestion
  "js_of_ocaml-compiler" {with-test & >= "4.1.0" < "7.0.0"}
```